### DISCUSSION TITLE
No Airplane in Simulator

### DISCUSSION THREAD

**Discussion by ZerkratztTV** [flybywiresim/aircraft#2360] [Help]
In the MSFS theres just the airplane but no graphics like no cockpit, nothing just some lights but not more. It looks like it's vanished. Any advices? Yes I can start it, but it's just not there.

Also if I delete the mod, the basic A320 isn't there, like not in the Airplanes-List.

Thanks for help.

**Comment by MapleNAV-TCASFAULT** [comment]
Go to content manager in profile in MSFS and download the possibly missing default a320.
